src/main/java/frc/robot/commands/StopDrive.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class StopDrive extends CommandBase {
```

Comment:
Command class names should end with `Command` (so `StopDrive` -> `StopDriveCommand`)